sdk/storage/inc/blobs/unused/internal/protocol/BlobRestClient.hpp

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include "azure/core/http.hpp"
+
+namespace Azure { namespace Storage { namespace Blobs {
+
+class BlobRestClient
+{
+public:
+    class Container
+    {
+        public:
+            
+        static Azure::Core::Http::Request Create_CreateMessage (
+            const Azure::Core::Http::Pipeline& pipeline,
+            const Azure::Core::Http::Uri& resourceUri,
+            const PublicAccessType& accessType,
```

Comment:
I think we said that the only positional params are pipeline and url; ALL operation-specific parameters will be struct fields.